Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM ubuntu:18.10
+
+RUN apt-get -qq update \
+&& apt-get -qq -y install curl \
+&& curl -L https://raw.githack.com/stoplightio/prism/master/install | sh
```

Comment:
This needs a discussion. What should our Docker image be based on? The binaries we provide or the npm package?

If it's the former, we're going to need node as base image, otherwise we can probably just go with the scratch one.